pentaho-js/src/main/javascript/cdf-legacy/cdf-base.js

```diff
@@ -193,3 +195,51 @@ wd.cdf.endpoints = {
     return enc;
   }
 };
+pho = typeof pho !== "undefined" ? pho : {};
+pho.csrfUtil = (function() {
+  return {
+    getToken: function(url) {
+      if(!url) {
+        throw new Error("Argument 'url' is required.");
+      }
+      var FULL_QUALIFIED_URL = wd.cdf.endpoints.getWebappFullUrl() + "/";
+
+      // Check if the request will be for the pentaho application.
+      if(url.indexOf(FULL_QUALIFIED_URL) !== 0) {
```

Comment:
Please add an empty line after the closing `}`, before `var FULL_QUALIFIED_URL ...`.

Reply:
addressed the comments